timetracker/entries/models.py

```diff
@@ -2,13 +2,51 @@
 
 from django.utils import timezone
 
+class Country(models.Model):
+	name = models.CharField(max_length=100, null=True)
+	
+	class Meta:
+		verbose_name_plural = "Countries"
+	
+	def __str__(self):
+		return self.name
+
+class Address(models.Model):
+	unit_no = models.CharField(max_length=15, default=None, blank=True, null=True)
```

Comment:
We haven't covered it in class, but generally you're want to use `blank=True, default=''` on `CharFields` that you're happy to allow to be empty.